src/objc_stubs.c

```diff
@@ -10,11 +10,12 @@
  */
 
 #include "arraytypes.h"
-#include "class.c"
+#include "aggregate.h"
 #include "mars.h"
 #include "objc.h"
```

Comment:
objc.h currently does not compile as is, it depends on other includes before being included. I think you should add some more of the forward declarations in objc.h.